pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -598,3 +598,28 @@ def test_getitem_loc_commutability(multiindex_year_month_day_dataframe_random_da
     result = ser[2000, 5]
     expected = df.loc[2000, 5]["A"]
     tm.assert_series_equal(result, expected)
+
+
+def test_getitem_loc_datetime():
```

Comment:
put partial in the name of the test, move to pandas/tests/indexing/test_partial.py

Reply:
Done